index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Miftah's Protfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;500&family=Roboto&display=swap"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"/>
+</head>
+<body>
+    <header>
+        <a href="#" class="mylogo">Miftah</a>
+        <nav class="mob-version">
+           <i class="fa fa-bars">
+
```

Comment:
- [ ] Please, try to avoid empty lines or blank spaces, as this is part of the [HTML && CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). Please make sure to check it on the entire project.

Reply:
@Lordkaito Thank you so much for reviewing my code, Yes I noticed this mistake, I have reviewed the entire project and removed extra space.